.circleci/config.yml

```diff
@@ -30,3 +30,17 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+
+      - architect/push-to-app-collection:
+          context: "architect"
+          name: aws-app-collection
+          app_name: "tempo"
+          app_namespace: "tempo"
+          app_collection_repo: "aws-app-collection"
+          requires:
+            - "package and push tempo chart to control-plane-catalogs"
```

Comment:
This dependency is probably broken: I didn't find this stage in the existing `config.yml`.

Reply:
You're right I fixed it